akka-http-core/src/main/resources/reference.conf

```diff
@@ -431,6 +431,14 @@ akka.http {
 
       # Enable verbose debug logging for all ingoing and outgoing frames
       log-frames = false
+
+      # When there is no data transmitted, send a HTTP/2 ping frame with this interval to make sure the connection is
+      # kept alive, 0 disables sending ping frames. Must be full seconds.
+      ping-interval = 0s
```

Comment:
should we name this `keepalive-time` to be consistent with https://github.com/grpc/grpc/blob/master/doc/keepalive.md ?

Reply:
Hmm, I thought about that but that it also applies when not using gRPC and the HTTP/2 term is ping/pong?

Otoh keepalive is the purpose more than the underlying terminology so maybe that is better.

Reply:
Updated to use keepalive as name for it.